common/snapshots.py

```diff
@@ -2165,7 +2164,7 @@ def fileInfo(self):
                         continue
                     info = line[:index].strip().split(' ')
                     if len(info) == 3:
-                        d[f] = (int(info[0]), info[1], info[2]) #perms, user, group
+                        d[f.encode('utf-8','replace')] = (int(info[0]), info[1].encode('utf-8','replace'), info[2].encode('utf-8','replace')) #perms, user, group
```

Comment:
It's pretty much like `str`. You just need to stay with one of each. `b'foo bar'.split(' ')` for example wouldn't work but `b'foo bar'.split(b' ')`. To convert between both you have `str.encode()` and `bytes.decode()`. That's it. But I was also confused by `bytes` after I switched BIT from Python2 to Python3.